alvr/server_openvr/cpp/platform/linux/EncodePipelineNvEnc.cpp

```diff
@@ -63,11 +63,12 @@ alvr::EncodePipelineNvEnc::EncodePipelineNvEnc(
     Renderer* render,
     VkContext& vk_ctx,
     VkFrame& input_frame,
-    VkFrameCtx& vk_frame_ctx,
+    VkImageCreateInfo image_create_info,
     uint32_t width,
     uint32_t height
 ) {
     r = render;
+    alvr::VkFrameCtx vk_frame_ctx(vk_ctx, image_create_info);
```

Comment:
Basically the important part is that this gets stored as long as the class exists because it will destroy that internal ffmepg frame context when it gets destructed. Which means it has to become a member of the struct. (And because lifetimes are finnicky directly below the renderer member). Sorry if my comments have been a bit confusing, only C++ calls them member variables (and I shortened that even further), other languages call it attributes or something.